examples/example-auth/pages/index.tsx

```diff
@@ -8,7 +8,7 @@ export default function IndexPage() {
   return (
     <>
       <Head>
-        <title>Next.js for Drupal | Authentication Example</title>
+        <title key="title">Next.js for Drupal | Authentication Example</title>
```

Comment:
This code example is from the [docs on Next.js that mention keys]():

> To avoid duplicate tags in your head you can use the key property, which will make sure the tag is only rendered once, as in the following example:
> 
> ```js
> function IndexPage() {
>   return (
>     <div>
>       <Head>
>         <title>My page title</title>
>         <meta property="og:title" content="My page title" key="title" />
>       </Head>
>       <Head>
>         <meta property="og:title" content="My new title" key="title" />
>       </Head>
>       <p>Hello world!</p>
>     </div>
>   )
> }
> ```

You can see that don't put a `key` on the title tag.

Reply:
Agreed, but not for the docs, but for the function behind deduplication of the `<title>` Tag in the actual Head component here: https://github.com/vercel/next.js/blob/c5b5b1e3a3d85d498770a974c250c864e8d2a761/packages/next/src/shared/lib/head.tsx#L84

As per the docs, we could argue that only a single `<title>` is declared and deduplication is therefore not strictly necessarry.
However, the actual NextJS `<Head>` component handles the `<title>` and `<base>` tag differently from the `<meta>` tag. I guess, my initial problem originates in line 106 of the head component which sets the `isUnique` variable to false if a meta-property is not "name"  or has no key.
https://github.com/vercel/next.js/blob/c5b5b1e3a3d85d498770a974c250c864e8d2a761/packages/next/src/shared/lib/head.tsx#L106

It was an interesting dive into the head component,
Cheers